### PR TITLE
Add assembly case blocks

### PR DESCRIPTION
**Pull request by fvictorio**
Depends on #91.

### CODE REVIEW

**Comment by mattiaerre** [comment]
@fvictorio can you resolve the conflicts so that I can merge? thanks

**Comment by fvictorio** [comment]
@mattiaerre Fixed!

File: tests/Assembly/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -20,6 +32,18 @@ contract Assembly {
       }
     }
   }
+
+  function caseAssembly() {
+    assembly {
+      switch value
```
Comment (mattiaerre) [inline tests/Assembly/__snapshots__/jsfmt.spec.js.snap]: uhm, shall we have 2 spaces to indent the "case" inside the switch?

Comment (fvictorio) [reply tests/Assembly/__snapshots__/jsfmt.spec.js.snap]: You mean that the cases should be more indented than the switch? Yeah, I agree with that, and that's what prettier does in javascript.

Comment (fvictorio) [reply tests/Assembly/__snapshots__/jsfmt.spec.js.snap]: Done.

Comment (mattiaerre) [reply tests/Assembly/__snapshots__/jsfmt.spec.js.snap]: awesome! thanks so much

**Pull request merged**